src/talosctl/install.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+set -e
+
+source ./library_scripts.sh
+
+# nanolayer is a cli utility which keeps container layers as small as possible
+# source code: https://github.com/devcontainers-contrib/nanolayer
+# `ensure_nanolayer` is a bash function that will find any existing nanolayer installations,
+# and if missing - will download a temporary copy that automatically get deleted at the end
+# of the script
+ensure_nanolayer nanolayer_location "v0.5.6"
+
+# Example nanolayer installation via devcontainer-feature
+$nanolayer_location \
+    install \
+    devcontainer-feature \
+    "ghcr.io/devcontainers-extra/features/gh-release:1" \
+    --option repo='siderolabs/talos' --option binaryNames='talosctl' --option version="$VERSION"
+
```

Comment:
Could you please add this additional option?
`--option releaseTagRegex='^(?!.*(alpha|beta)).*$'`

This would filter out GitHub releases that contain "alpha" or "beta" in their name. Right now the latest release of `siderolabs/talos` is [v1.9.0-alpha.0](https://github.com/siderolabs/talos/releases/tag/v1.9.0-alpha.0), so it's installed by default. I would like to avoid installing not stable versions

I want to add such a filter at nanolayer level, but that's something for the future (https://github.com/devcontainers-extra/nanolayer/issues/1)